requirements.txt

```diff
@@ -1,3 +1,3 @@
 pymongo==2.7.1
 wheel==0.23.0
-
+numpy==1.9
```

Comment:
This is a bit of a heavy requirement we're bringing in here.  Is there a reasonable way to do the base36 encoding without numpy?